cachecontrol/controller.py

```diff
@@ -315,16 +330,25 @@ def cache_response(self, request, response, body=None,
                 self.cache.set(
                     cache_url,
                     self.serializer.dumps(request, response, body=body),
+                    expires=cc['max-age'],
                 )
 
             # If the request can expire, it means we should cache it
             # in the meantime.
             elif 'expires' in response_headers:
                 if response_headers['expires']:
-                    logger.debug('Caching b/c of expires header')
+
```

Comment:
Let's skip the extra whitespace in order to keep the if block connected with the following code.